tasks/zadanie_8.php

```diff
@@ -13,6 +13,8 @@
 $role = $params[0]; // tej linijki nie ruszamy :)
 $isLogged = $params[1]; // tej linijki nie ruszamy :)
 
-// ...
-
-echo "Dostęp przyznany";
\ No newline at end of file
+if ($role == "admin" && $isLogged == true) {
+        echo "Dostęp przyznany".PHP_EOL;
+} else {
+    echo "Brak dostępu".PHP_EOL;
+};
```

Comment:
tutaj bez średnika ;) 
```suggestion
}
```